python_modules/dagster/dagster/_core/storage/alembic/versions/043_add_run_tags_run_id_index.py

```diff
@@ -0,0 +1,45 @@
+"""add_run_tags_run_id_index
+
+Revision ID: 284a732df317
+Revises: 46b412388816
+Create Date: 2024-07-10 09:35:20.215174
+
+"""
+
+from alembic import op
+from dagster._core.storage.migration.utils import has_index, has_table
+
+# revision identifiers, used by Alembic.
+revision = "284a732df317"
+down_revision = "46b412388816"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    if not has_table("run_tags"):
+        return
+
+    if not has_index("run_tags", "idx_run_tags_run_idx"):
+        op.create_index(
+            "idx_run_tags_run_idx",
+            "run_tags",
+            ["run_id", "id"],
+            unique=False,
+            postgresql_concurrently=True,
+            mysql_length={
+                "pipeline_name": 512,
```

Comment:
"run_id": 255

Reply:
Thanks, I completely missed that one. Looking into it, it makes sense. I have applied the suggested changes.